templates/_helpers.tpl

```diff
@@ -30,6 +30,19 @@ stackstorm
 {{- end -}}
 {{- end -}}
 
+# Generate Docker image repository for st2actionrunner: Private 'docker.stackstorm.com' for Enterprise vs Public Docker Hub 'stackstorm' for FOSS version
+{{- define "st2actionrunnerImageRepository" -}}
```

Comment:
For simplicity reasons, it would be easier to avoid custom template for st2actionrunner, if possible. I saw the previous version, wondering if that worked good?